controls/pcidss_4.yml

```diff
@@ -0,0 +1,3297 @@
+policy: PCI-DSS
+title: Configuration Recommendations of a GNU/Linux System
+id: pcidss_4
+version: '4'
+source: https://docs-prv.pcisecuritystandards.org/PCI%20DSS/Standard/PCI-DSS-v4_0.pdf
+levels:
+- id: base
+
+controls:
+  - id: Req-1.1
+    title: '1.1 Processes and mechanisms for installing and maintaining network security
```

Comment:
I think there is no need to repeat the reference number in the title as well. Let's remove it from every item. I guess rerunning the script with a few adjustments should do the trick.